trollmoves/tests/test_client.py

```diff
@@ -976,16 +959,15 @@ def test_read_config(client_config_1_item):
     section_keys = conf[section_name].keys()
     for key in ["delete", "working_directory", "compression",
                 "heartbeat", "req_timeout", "transfer_req_timeout",
-                "nameservers", "providers", "topic", "publish_port",
-                "create_target_directory"]:
+                "nameservers", "providers", "topic", "publish_port", ]:
         assert key in section_keys
     assert isinstance(conf[section_name]["providers"], list)
 
 
 @patch('trollmoves.client.request_push')
 @patch('trollmoves.client.create_publisher_from_dict_config')
 @patch('trollmoves.client.Listener')
-def test_reload_config_single_chain(Listener, create_publisher_from_dict_config, request_push, client_config_1_item):
+def test_reload_config_single_chain(Listener, NoisyPublisher, request_push, client_config_1_item):
```

Comment:
Why are these renamed to `NoisyPublisher`?

Reply:
that's a miss, fixing

Reply:
done